### ISSUE TITLE
Investigate errors in recent travis runs

### ISSUE THREAD

**Issue by scullxbones** [scullxbones/akka-persistence-mongo#119]
From recent travis test logs:

```
java.lang.IllegalStateException: onError must not be called after onComplete
	at akka.stream.actor.ActorPublisher$class.onError(ActorPublisher.scala:245)
	at akka.contrib.persistence.mongodb.CurrentAllPersistenceIds.onError(RxMongoReadJournaller.scala:141)
	at akka.stream.actor.ActorPublisher$class.onErrorThenStop(ActorPublisher.scala:262)
	at akka.contrib.persistence.mongodb.CurrentAllPersistenceIds.onErrorThenStop(RxMongoReadJournaller.scala:141)
	at akka.contrib.persistence.mongodb.IterateeActorPublisher$$anonfun$defaults$1.applyOrElse(RxMongoReadJournaller.scala:75)
...
```

**Comment by scullxbones** [comment]
Another
```
16:09:42.015 [unit-test-akka-contrib-persistence-dispatcher-32] ERROR akka.contrib.persistence.mongodb.RxMongoPersistenceExtension$Configured$$anon$1 - Error bulk inserting into realtime collection
reactivemongo.core.actors.Exceptions$ClosedException$: MongoError['This MongoConnection is closed']
	at reactivemongo.core.actors.Exceptions$ClosedException$.<clinit>(actors.scala) ~[reactivemongo_2.11-0.11.9.jar:0.11.9]
	at reactivemongo.core.actors.MongoDBSystem$$anonfun$4$$anonfun$applyOrElse$8.apply(actors.scala:310) ~[reactivemongo_2.11-0.11.9.jar:0.11.9]
	at reactivemongo.core.actors.MongoDBSystem$$anonfun$4$$anonfun$applyOrElse$8.apply(actors.scala:308) ~[reactivemongo_2.11-0.11.9.jar:0.11.9]
	at scala.collection.mutable.LinkedHashMap.foreach(LinkedHashMap.scala:139) ~[scala-library-2.11.7.jar:1.0.0-M1]
	at reactivemongo.core.actors.MongoDBSystem$$anonfun$4.applyOrElse(actors.scala:308) ~[reactivemongo_2.11-0.11.9.jar:0.11.9]
```

**Comment by JeanFrancoisGuena** [comment]
Hello,

I fixed the first error and will submit you a small PR for that in a few moments...

The second one does not appear anymore, it was bound to the `doBatchAppend` method we decided to remove (remember, I asked you to choose between two implementations for this method on RxMongo side when submitting the https://github.com/scullxbones/akka-persistence-mongo/pull/118 PR)

**Comment by scullxbones** [comment]
Closed by PR #120
